db/migrate/20160902111740_initial_schema.rb

```diff
@@ -11,8 +11,8 @@
 
       column :url_path, String, unique: true
 
-      column :name, String
       column :slug, String, unique: true
+      column :real_name, String, null: true
```

Comment:
What do you think about renaming this column to `display_name`, as some people might not want to use their real name (so this column would not always include the real name)?